api/coreservice.go

```diff
@@ -62,6 +67,11 @@ import (
 )
 
 const _workerNumbers int = 5
+const (
+	// defaultTraceTimeout is the amount of time a single transaction can execute
+	// by default before being forcefully aborted.
+	defaultTraceTimeout = 5 * time.Second
```

Comment:
Why defaultTraceTimeout is only used in `TraceTransaction`, but not in `TraceCall`?